monitors/src/test/java/MonitorTests.java

```diff
@@ -0,0 +1,91 @@
+package coppercore.monitors.test;
+
+import coppercore.monitors.Monitor;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class MonitorTests {
+    // TODO: FIGURE OUT HOW TO TEST THIS
+    // Callbacks are hard :(
```

Comment:
could you make some variable owned by the test fixture and then increment it in the callback function? Or do something with a static variable?

Reply:
Done! Thanks for the suggestion on how to do this.